tests/test_functions.py

```diff
@@ -14,10 +14,72 @@ def test_Constant():
     constant1.differentiate()
     assert constant1.value == 0
 
-    constant2 = Constant(5)
+    constant2 = Constant(5, 2)
     constant2.integrate('x')
     assert isinstance(constant2, Variable)
-    assert constant2.__str__() == "5{x}"
+    assert constant2.__str__() == "25{x}"
+
+    # Tests for Operator Overloaded functions.
+
+    # Addition & Subtraction
+
+    constant4 = Constant(2, 2)
+    constant5 = Constant(7)
+    constant3 = constant4 - constant5
+    assert constant3.__str__() == "{-3}"
+
+    constant4 = Constant(7, 2)
+    constant0 = Constant(5)
+    constant6 = constant4 - constant3 - constant5 + constant0
+    assert constant6.__str__() == "{50}"
+
+    constant0 = Constant(5, 2, 2)
+    constant2 = constant0
+    variable0 = Variable(5, 'X', 3)
+    summation0 = constant0 - variable0 + constant2
+    assert summation0.__str__() == "{({100}-5{X}^{3})}"
+
+    constant0 = Constant(5, 2, 2)
+    constant2 = constant0
+    variable0 = Variable(5, 'X', 3)
+    summation0 = constant0 + variable0 + constant2
+    assert summation0.__str__() == "{({100}+5{X}^{3})}"
+
+    # Multiplication & Division
+
+    constant0 = Constant(0, 2)
+    constant1 = Constant(5)
+    mul1 = constant0 * constant1
+    assert mul1.calculate() == 0
+    mul1 = constant1 * constant0
+    assert mul1.calculate() == 0
+    mul1 = constant1 * constant1 + constant0 * constant1 + constant0 * constant1
+    assert mul1.calculate() == 25
+
+    constant1 = Constant(5, 2)
+    constant2 = Constant(4, 2)
+    variable0 = Variable(3, 'X', 3)
+    mul3 = constant1 * (constant2 + variable0)
+    assert mul3.__str__() == "{({400}+75{X}^{3})}"
+
+    constant1 = Constant(5, 2)
+    constant2 = Constant(4, 2)
+    variable0 = Variable(3, 'X', 3)
+    mul3 = constant1 / (constant2 + variable0)
+    assert mul3.__str__() == "{25}*{({16}+3{X}^{3})}^{-1}"
+
+    constant1 = Constant(3, 2)
+    constant2 = Constant(4, 2)
+    variable0 = Variable(3, 'X', 3)
+    mul3 = constant1 - constant1/(constant2/variable0 + constant1)
```

Comment:
By _Expression_, I meant the `visma.functions.structure.Expression` class.
```suggestion
    mul3 = constant1 - Expression([constant2, Divide(), variable0, Plus(), constant1]);
```